esque/cli/commands.py

```diff
@@ -311,10 +311,21 @@ def transfer(
             return
 
         state.config.context_switch(to_context)
-        _produce_from_file(topic, to_context, working_dir, avro)
+        _produce_from_files(topic, to_context, working_dir, avro)
 
 
-def _produce_from_file(topic: str, to_context: str, working_dir: pathlib.Path, avro: bool):
+@esque.command("produce", help="Produce messages from directory based on output from transfer command")
+@click.argument("topic", required=True)
+@click.option(
+    "-d", "--directory", help="Sets the directory that contains Kafka messages", type=click.STRING, required=True
+)
+@click.option("-a", "--avro", help="Set this flag if the topic contains avro data", default=False, is_flag=True)
```

Comment:
What happens if I forget to set this flag but the data is avro? Wouldn't it be possible to detect avro or not automatically based on the first few lines?

Reply:
This is definitely better then using this flag. But I would change this in another branch.

Reply:
Fair. Created an issue for it: https://github.com/real-digital/esque/issues/41